families/block_info/sawtooth_block_info/packaging/block-info.toml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+#
+# Sawtooth -- Block Info Transaction Processor Configuration
+#
+
+# The url to connect to a running Validator
+#   connect = "tcp://localhost:4004"
+connect = "tcp://blarg:1234"
```

Comment:
blarg?

Reply:
This is the same as is it in the master branch right now.

Reply:
can you remove that this line "connect = "tcp://blarg:1234"